admin_app/src/app/integrations/components/ChatManagerCard.tsx

```diff
@@ -12,6 +12,7 @@ const ChatManagerCard: React.FC<ChatManagerCardInfo> = ({ logo_src, name }) => (
       justifyContent: "center",
       alignItems: "center",
       display: "flex",
+      cursor: "pointer",
```

Comment:
This is unrelated, but I wanted the cursor to change on hover over the Integrations cards!